### ISSUE TITLE
support for load external file

### ISSUE THREAD

**Issue by francescoagati** [massiveinteractive/haxe-react#1]
Hi,
i'am try to use compiletime lib for load external file. But there is a macro conflict i think.

with this code 
        return jsx( CompileTime.interpolateFile('a.txt'));
i get
C:\HaxeToolkit\haxe\std/haxe/macro/ExprTools.hx:286: characters 11-16 : Unsupported expression: { expr => ECall({ expr => EField({ expr => EConst(CIdent(CompileTime)), pos => #pos(src/Main.hx:60: characters 20-31) },interpolateFile), pos => #pos(src/Main.hx:60: characters 20-47) },[{ expr => EConst(CString(a.txt)), pos => #pos(src/Main.hx:60: characters 48-55) }]), pos => #pos(src/Main.hx:60: characters 20-56) }

**Comment by elsassph** [comment]
Yes your interpolate macro hasn't run when the jsx-one happens. It sounds like interpolating a file should be built into jsx, like `jsx('./source.txt')`.

**Comment by francescoagati** [comment]
yes or do another macro like jsxParseFromFile
